backend/yugioh/admin.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib import admin
+
+from .models import YugiohCard, YugiohCardInSet, YugiohCardRarity, YugiohCardSet
+
+
+@admin.register(YugiohCard)
+class YugiohCardAdmin(admin.ModelAdmin):
+    list_display = ['type', 'frame_type', 'description', 'attack', 'defense', 'level', 'race', 'attribute', 'archetype']
+    list_filter = ['type', 'frame_type', 'description', 'attack', 'defense', 'level', 'race', 'attribute', 'archetype']
+    search_fields = ['type', 'frame_type', 'description', 'attack', 'defense', 'level', 'race', 'attribute', 'archetype']
+
+
+@admin.register(YugiohCardInSet)
+class YugiohCardInSetAdmin(admin.ModelAdmin):
```

Comment:
- [ ] Add the card name as the first column

Reply:
The name was added